tests/bootstrap.php

```diff
@@ -7,4 +7,3 @@
 
 // autoload abstract TestCase classes in test directory
 $autoloader->add('Omnipay', __DIR__.'/../vendor/omnipay/omnipay/tests');
-
```

Comment:
I don't think this will be compliant with a style check (requires blank end of line)

Reply:
I ran the code through phpcbf fix PSR2 before committing

Reply:
https://www.php-fig.org/psr/psr-2/#22-files

Might be bug there, then.

Reply:
Actually, my editor shows a blank line, so maybe this is a line ending change or something

Reply:
Travis will also run PSR2 standard check over it

Reply:
Weird. Maybe there was an extra line or something? I now we did a bunch of PSR updates and things for the 2.0 upgrade.